portsyncd/portsyncd.cpp

```diff
@@ -179,7 +195,9 @@ static void notifyPortConfigDone(ProducerStateTable &p)
 
 bool handlePortConfigFromConfigDB(ProducerStateTable &p, DBConnector &cfgDb, bool warm)
 {
-    cout << "Get port configuration from ConfigDB..." << endl;
+    SWSS_LOG_ENTER();
+
+    SWSS_LOG_NOTICE("Getting port configuration from ConfigDB...");
```

Comment:
Why change this part? This is one time `cout` at the beginning, no harm to cout.

Reply:
SWSS_LOG_* is a usual way to do logging in SONiC, why leave it to be cout?

Reply:
It is always good  to improve the code, but I feel we should keep consistency. If we are changing the style in this function that is not touched by your core logic, we should make other places in this file to be consistent.  like line 112 etc.  Or we could leave all these improvements in a separate PR, and address the whole file. This way it would be easier to maintain commits across branches etc.